packages/babel-plugin/src/utils/css-builders.tsx

```diff
@@ -550,87 +624,100 @@ const extractTemplateLiteral = (node: t.TemplateLiteral, meta: Metadata): CSSOut
   const variables: CSSOutput['variables'] = [];
 
   // Quasis are the string pieces of the template literal - the parts around the interpolations
-  const literalResult = node.quasis.reduce<string>((acc, quasi, index): string => {
-    const nodeExpression = node.expressions[index] as t.Expression | undefined;
+  const literalResult = node.quasis.reduce<string>(
+    (acc: string, quasi: t.TemplateElement, index: number): string => {
+      let nodeExpression = node.expressions[index] as t.Expression | undefined;
 
-    if (
-      !nodeExpression ||
-      (t.isArrowFunctionExpression(nodeExpression) && t.isLogicalExpression(nodeExpression.body))
-    ) {
-      const suffix = meta.context === 'keyframes' ? '' : ';';
-      return acc + quasi.value.raw + suffix;
-    }
+      if (
+        !nodeExpression ||
+        (t.isArrowFunctionExpression(nodeExpression) && t.isLogicalExpression(nodeExpression.body))
+      ) {
+        const suffix = meta.context === 'keyframes' ? '' : ';';
+        return acc + quasi.value.raw + suffix;
+      }
 
-    const { value: interpolation, meta: updatedMeta } = evaluateExpression(nodeExpression, meta);
+      // Deal with any negative values such as:
+      // margin: -${gridSize}, top: -${gridSize} etc.
+      if (quasi.value.raw.endsWith('-') && !t.isArrowFunctionExpression(nodeExpression)) {
+        quasi.value.raw = trimUnaryOperator(quasi.value.raw);
```

Comment:
I would just inline this and add a comment above it, i.e.

```
// Remove the '-' from the quasi, as it will soon be moved to a unary expression
quasi.value.raw = quasi.value.raw.slice(0, -1).trim();